tests/lib.rs

```diff
@@ -175,6 +175,7 @@ fn lock_and_source_clean() -> io::Result<()> {
     Ok(())
 }
 
+#[cfg(not(target_os = "windows"))]
```

Comment:
If possible we should create a test which tests the same thing on windows. The test just needs to get an IO error when attempting to clean one the files.